qualtran/bloqs/util_bloqs.py

```diff
@@ -113,7 +113,7 @@ def wire_symbol(self, soq: 'Soquet') -> 'WireSymbol':
         if soq.reg.shape:
             text = f'[{", ".join(str(i) for i in soq.idx)}]'
             return directional_text_box(text, side=soq.reg.side)
-        return directional_text_box(' ', side=soq.reg.side)
+        return directional_text_box('S', side=soq.reg.side)
```

Comment:
motivate this change? presumably the splitting action should be implied by the circuit topology

Reply:
Using `symbol.text` leads to an empty symbol which `qpic` doesn't like for labels of the gate. I can probably hack around if needed